### DISCUSSION TITLE
browser.getPageLogs is not a function

### DISCUSSION THREAD

**Discussion by wangli-wangli** [webdriverio/webdriverio#6830] [Q&A]
![image](https://user-images.githubusercontent.com/33982322/117233736-49973980-ae56-11eb-80cc-38ab7bdf1040.png)
 browser.getPageLogs is not a function。How to get page log？

**Comment by christian-bromann** [comment]
As mentioned in the docs:

> All commands are only supported on Chrome using Sauce Labs Extended Debugging capabilities. You can enable these by setting the following Sauce options:
> ```    
>    {
>         browserName: 'Chrome',
>         browserVersion: 'latest',
>         platformName: 'Windows 10',
>         'sauce:options': {
>             extendedDebugging: true
>         }
>    }
> ```
